components/event-publisher/event-output-adapters/org.wso2.carbon.event.output.adapter.rdbms/src/main/java/org/wso2/carbon/event/output/adapter/rdbms/RDBMSEventAdapter.java

```diff
@@ -324,6 +324,7 @@ public void executeDbActions(Object message)
                     stmt = con.prepareStatement(executionInfo.getPreparedUpdateStatement());
                     populateStatement(map, stmt, executionInfo.getUpdateQueryColumnOrder());
                     int updatedRows = stmt.executeUpdate();
+                    con.setAutoCommit(false);
```

Comment:
Shouldn't con.setAutoCommit(false) be called before we prepare the statement and execute it. If we set autoCommit to false just before committing, at the time of doing stmt.executeUpdate(), autoCommit is still set to true. Wouldn't that mean that at the time of executing the statement, there is still a possibility of autoCommit being triggered?